daemons/gptp/common/ieee1588port.cpp

```diff
@@ -886,6 +898,17 @@ void IEEE1588Port::processEvent(Event e)
 		setAsCapable(false);
 		pdelay_count = 0;
 		break;
+
+	case PDELAY_RESP_PEER_MISBEHAVING_TIMEOUT_EXPIRES:
+		XPTPD_INFO("Timeout expired! Restarting PDelay");
```

Comment:
Might want to explicitly say which timeout expired, in case someone is looking through logs to investigate "odd" behavior.